spec/lcm/integration/help_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'gooddata'
+
+describe 'HelpBrick' do
+  it "print help" do
+    @brick_result = GoodData::Bricks::Pipeline.help_brick_pipeline.call({})
+    expect(@brick_result[:results]["Help"].map { | brick | brick[:available_brick] }).to eq(
+      %w[ release users rollout provisioning user_filters help hello_world ]
```

Comment:
warning: Layout/SpaceInsidePercentLiteralDelimiters: Do not use spaces inside percent literal delimiters. (rubocop)